src/main/java/org/jenkins/plugins/lockableresources/LockableResourcesManager.java

```diff
@@ -8,6 +8,7 @@
  * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * * */
 package org.jenkins.plugins.lockableresources;
 
+import com.iwombat.util.GUIDUtil;
```

Comment:
What's this? Is it even used?

Reply:
Ah, that is probably a leftover before I used the standard java UUID generator. I have removed it.